### PR TITLE
Suggest to modify the VLAN interface name when the VLAN ID is modified

### PR DESCRIPTION
**Pull request by teclator**
## Problem

The VLAN ID and the interface name are configured in different tabs, thus, it is very common to have a misleading interface name when the VLAN ID is modified.

- https://bugzilla.suse.com/show_bug.cgi?id=1174363

## Solution

Suggest a interface name based on the new 'VLAN ID' when it is modified.

## Screenshot

SuggestVlanInterfaceName
ModifiedInterfaceName

### CODE REVIEW

File: src/lib/y2network/widgets/vlan_id.rb [hunk 1]
```diff
@@ -52,6 +55,25 @@ def minimum
...
+      end
+
+      def suggested_name
+        "vlan#{value}"
+      end
+
+      def suggest_vlan_name
+        Yast::Popup.YesNo(
+          format(
+            _("Would you like to adapt the interface name from '%s' to '%s'?"),
```
Comment (mchf) [inline src/lib/y2network/widgets/vlan_id.rb:72]: I'd add some more noise why the change is being suggested. Something like "VLAN with id  20 is being defined. Would you mind to modify device name from vlan0 to vlan20"

**Review by mchf — APPROVED** [review]
only one NP which i don't insist on ;-) lgtm

File: src/lib/y2network/widgets/vlan_id.rb (hunk 1 above)
Comment (teclator) [reply src/lib/y2network/widgets/vlan_id.rb:72]: Screenshot from 2021-01-25 08-27-58
